### PR TITLE
Remove contributors list from template, fix template markup and update README files

### PR DESCRIPTION
**Pull request by igloosi**
Contributors list is already out of date. We will need to explore a different way of crediting contributions.

This PR:
* removes the [list from the component ](https://trello.com/c/ZpBUWeym/263-remove-the-contributors-list-from-the-individual-component-readmes)view template
* updates template so no wrapper divs are present in the generated markdown files
* updates `pre` and `code` blocks in generated markdown files
* updates all the README files with latest changes

### CODE REVIEW

File: src/components/back-link/README.md [hunk 1]
```diff
@@ -1,3 +1,7 @@
+<div class="govuk-o-width-container">
```
Comment (36degrees) [inline src/components/back-link/README.md]: 🤔

File: src/components/back-link/README.md [hunk 2]
```diff
@@ -38,15 +42,17 @@ To consume the back-link component you must be running npm version 5 or above.
 
 When compiling the Sass files you'll need to define includePaths to reference the node_modules directory. Below is a sample configuration using gulp
 
-      .pipe(sass({
-          includePaths: 'node_modules/'
-      }))
+<pre>  `.pipe(sass({
```
Comment (36degrees) [inline src/components/back-link/README.md]: Where did these `pre` tags come from?

**Comment by igloosi** [comment]
has there been work on the component template?

**Comment by 36degrees** [comment]
Possibly related to https://github.com/alphagov/govuk-frontend/commit/4c01202842352f6f8488065cab4d9dd1db976b1c#diff-841b0a984418ab47f365fb13d1a0ac89?

**Comment by igloosi** [comment]
ok will fix that first then and check markdown compiler

**Comment by igloosi** [comment]
updated with template fixes

**Comment by igloosi** [comment]
rebased with #405

File: src/components/breadcrumbs/README.md [hunk 3]
```diff
@@ -332,39 +324,12 @@ If you are using Nunjucks,then macros take the following arguments
...
 Below is an example setup using express configure views:
 
-<pre>  `nunjucks.configure('node_modules/@govuk-frontend`, {
-    autoescape: true,
-    cache: false,
-    express: app
-  })` 
-  </pre>
-
-## Getting updates
```
Comment (36degrees) [inline src/components/breadcrumbs/README.md]: All of the READMEs appear to be missing everything from here to the license, despite most of it still being in the template?

**Comment by igloosi** [comment]
yeah missing `</`

File: src/views/component.njk [hunk 4]
```diff
@@ -57,72 +56,59 @@
   <p class="govuk-body">{% block componentDependencies %}{% endblock %}</p>
 
   <h2 class="govuk-heading-l">Installation</h2>
-  <pre><code>npm install --save @govuk-frontend/{{ componentName }}</code></pre>
+
+  <code>npm install --save @govuk-frontend/{{ componentName }}</code>
```
Comment (36degrees) [inline src/views/component.njk]: What's the logic in `<pre><code>` vs `<code>`? Looking at the rendered readmes in GitHub, these no longer render as code blocks but as inline. Is that the desired outcome?

Comment (igloosi) [reply src/views/component.njk]: its a single line so doesnt stringly need pre. but can unify it

Comment (36degrees) [reply src/views/component.njk]: I think if it's a single line but not inline it should still be a code _block_.

I.e. the difference between `this` inline code and

```
this single line code block
```

Comment (igloosi) [reply src/views/component.njk]: ql, updated

File: src/views/index.njk [hunk 5]
```diff
@@ -1,7 +1,6 @@
 {% extends "layout-debug.njk" %}
 
 {% block content %}
-<div class="govuk-o-main-wrapper">
```
Comment (36degrees) [inline src/views/index.njk]: Why was this removed? There is now no space between the top of the page and the heading on the index page.

Comment (igloosi) [reply src/views/index.njk]: my bad, moved the other one into the logic but forgot to undelete this one.
